### PR TITLE
add doc for zkPass and Reclaim protocol

### LINKED ISSUE #232: Document zkpass and reclaim protocol usage
Create a new page `/extensions/credentials/zkpass` under `Extensions / Credentials` in the navigation menu.

Add documentation for developers that explain:

- [ ] How they can direct their users to obtain a zkpass proof. Show code examples that link to the deployed `proofs.verida.network` applciation.
- [ ] Example code showing how their application can request the zkpass proof and verify it

Do the same for reclaim protocol.

### PR DESCRIPTION
**Pull request by icetrust0212**
closes #232

### CODE REVIEW

File: docs/extensions/credentials/zkpass.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+Verifiers can also send you ZkPass proof requests. The Verida Wallet will automatically generate the Zero-Knowledge proof (ZKP) for you and send it to the Verifier. The Zero-Knowledge proof means no data is actually shared with the Verifier, only the fact that you have a valid credential satisfying the request.
+
+## Pre-requirement
+
+You need to install [ZkPass TransGate extension](https://zkpass.gitbook.io/zkpass/user-guides/transgate) to issue zkPass credentials.
+
+### Issuing a ZkPass credential
+
+The [proof-connector](https://proofs.verida.network/) app can issue ZkPass credential.
+You need to provide `veridaDid` where generated credential will go to in the url. For example, the url can be like [this](https://proof.verida.network/add-credential?veridaDid=did:vda:mainnet:0x378E209c8Cdc071b1ad7d0b4aBE300309A7bE541), or [this](https://proof.verida.network/add-credential?veridaDid=did:vda:mainnet:0x378E209c8Cdc071b1ad7d0b4aBE300309A7bE541&schemaId=ef39adb26c88439591279e25e7856b61).
```
Comment (tahpot) [inline docs/extensions/credentials/zkpass.md:30]: Show the actual URLs

File: docs/extensions/credentials/zkpass.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+It will redirects you to page where you select schemas.
```
Comment (tahpot) [inline docs/extensions/credentials/zkpass.md:32]: Make it clear users can request a schema in the URL.

List all the currently supported schemas.

File: docs/extensions/credentials/zkpass.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+Once you select schema from ZkPass protocol, you can start process to create credentials.
+
+![ZkPass credential issuer - Start generating](/img/extensions/zkpass/start-process.png)
+
+It will redirects you to the platform (For example: Binance) and once you click `Start` button in TransGate extension, the the process should start.
+![ZkPass credential issuer - TransGate](/img/extensions/zkpass/transgate.png)
+
+
+```
+export type ZkPassResult = {
```
Comment (tahpot) [inline docs/extensions/credentials/zkpass.md]: Does this code section need a heading and a description? I'm unclear why it's here.

File: docs/extensions/credentials/zkpass.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
+---
```
Comment (tahpot) [inline docs/extensions/credentials/zkpass.md:1]: I think this documentation should have a slightly different structure:

1. Requesting a zkPass credential (code example requesting a credential from the user's inbox)
2. Issuing a zkPass credential (code example, explain schemas, list all the schemas; make it clear this is only needed if the user doesn't have the credential)
3. Verifying a credential (code example to verify from an inbox response using the zkPass SDK, also explain and link to the proof connector verify page)

**Review by tahpot — CHANGES_REQUESTED** [review]
It should be `zkPass` not `ZkPass`.

`redirects` should be `redirect`

File: docs/extensions/credentials/zkpass.md (hunk 2 above)
Comment (icetrust0212) [reply docs/extensions/credentials/zkpass.md:32]: Done

File: docs/extensions/credentials/zkpass.md (hunk 1 above)
Comment (icetrust0212) [reply docs/extensions/credentials/zkpass.md:30]: Done

File: docs/extensions/credentials/zkpass.md (hunk 3 above)
Comment (icetrust0212) [reply docs/extensions/credentials/zkpass.md]: Done

File: docs/extensions/credentials/zkpass.md (hunk 4 above)
Comment (icetrust0212) [reply docs/extensions/credentials/zkpass.md:1]: Done

**Review by tahpot — APPROVED** [review]
LFG

**Pull request merged**